### ISSUE TITLE
Will the labels for test set be released?

### ISSUE THREAD

**Issue by XJay18** [yinanhe/ForgeryNet#14]
Currently, it seems that only the images/videos are available on your webpage. I wonder will the labels for the test set be released?

**Comment by Kuznetspa** [comment]
Hi! I have the same question. Will labels be published for the testset?

**Comment by YoshihikoFuruhashi** [comment]
I can't find the testset label either. 💦 
Does anyone have one?

**Comment by NeverMoreLCH** [comment]
same question

**Comment by shen8424** [comment]
It's hard to follow this work.
